FE/src/app/app.component.ts

```diff
@@ -1,11 +1,12 @@
-import { Component } from '@angular/core';
-import { RouterLink, RouterOutlet } from '@angular/router';
+import {Component} from '@angular/core';
+import {RouterLink, RouterOutlet} from '@angular/router';
 import {SharedModule} from './modules/shared/shared.module';
 
 @Component({
   selector: 'app-root',
   imports: [RouterLink, RouterOutlet, SharedModule],
   templateUrl: './app.component.html',
+  standalone: true,
```

Comment:
Do you still need it there? I think it is true by default in newer Angular versions.